settings.gradle.kts

```diff
@@ -14,7 +14,7 @@ dependencyResolutionManagement {
         maven("https://jitpack.io")
         maven("https://androidx.dev/storage/compose-compiler/repository/")
         // Hold until https://issuetracker.google.com/issues/325372937#comment4 is fixed
```

Comment:
This can be removed.

Reply:
Fixes will land on Gboard updates?

Reply:
Fixed by https://android-review.googlesource.com/2975591.